packages/versioning/src/versioning.ts

```diff
@@ -546,31 +529,45 @@ interface VersionProjections {
 /**
  * @internal
  */
-export function indexVersions(program: Program, versions: Map<Namespace, Version>) {
```

Comment:
Breaking change rename.

Reply:
not really as this is marked as `@internal`